src/java.base/share/classes/jdk/internal/PreviewFeature.java

```diff
@@ -62,6 +62,13 @@
         // JDK 15. Since the JDK 14 codebase uses the enum constant, it is
         // necessary for PreviewFeature in JDK 15 to declare the enum constant.
         TEXT_BLOCKS,
+        /*
```

Comment:
It doesn't really matter, but it would look more consistent if TEXT_BLOCKS and RECORDS used the same kind of comment syntax.

Reply:
thanks for the comment, I have uploaded commit [a037310](https://github.com/openjdk/jdk/pull/291/commits/a03731005bae069a18e4ef2f0dced7019c2a4081) to fix this